src/status_im/ui/screens/qr_scanner/views.cljs

```diff
@@ -12,7 +12,8 @@
             [status-im.ui.screens.qr-scanner.styles :as styles]
             [status-im2.config :as config]))
 
-(def camera (reagent/adapt-react-class CameraKitCamera))
+(def camera (reagent/adapt-react-class Camera))
```

Comment:
It looks like we can use the vars defined in `react-native.camera-kit`.

Reply:
I haven't updated this code because this is an old file/code/namespace. 
No worries, I will update it 👍

Reply:
Oh sorry, I missed that.